content/docs/contributor/anteaterapi/setup.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Setting up
```

Comment:
The title should be title case. In this case "Setting Up". I'll set up a linting rule for this in the future so this doesn't take up review time.